tests/integration/test_data/test_nitrate/integration.test_nitrate.NitrateExport.test_create.yaml

```diff
@@ -0,0 +1,290 @@
+_requre:
+  DataTypes: 1
+  key_strategy: StorageKeysInspectFull
+  version_storage_file: 3
+unittest.case:
+  integration.test_nitrate:
+    tmt.base:
+      tmt.export:
+        nitrate.containers:
+          xmlrpc.client:
+            requre.objects:
+              requre.cassette:
+                nitrate.xmlrpc_driver:
+                  single_request_with_cookies:
+                  - metadata:
+                      guess_type: Tuple
+                      latency: 0.9707729816436768
+                      module_call_list:
+                      - unittest.case
+                      - integration.test_nitrate
+                      - tmt.base
+                      - tmt.export
+                      - nitrate.containers
+                      - xmlrpc.client
+                      - requre.objects
+                      - requre.cassette
+                      - nitrate.xmlrpc_driver
+                      - single_request_with_cookies
+                    output:
+                    - []
+        nitrate.mutable:
+          nitrate.base:
+            nitrate.immutable:
+              nitrate.base:
+                nitrate.immutable:
+                  xmlrpc.client:
+                    requre.objects:
+                      requre.cassette:
+                        nitrate.xmlrpc_driver:
+                          single_request_with_cookies:
+                          - metadata:
+                              guess_type: Tuple
+                              latency: 0.8866176605224609
+                              module_call_list:
+                              - unittest.case
+                              - integration.test_nitrate
+                              - tmt.base
+                              - tmt.export
+                              - nitrate.mutable
+                              - nitrate.base
+                              - nitrate.immutable
+                              - nitrate.base
+                              - nitrate.immutable
+                              - xmlrpc.client
+                              - requre.objects
+                              - requre.cassette
+                              - nitrate.xmlrpc_driver
+                              - single_request_with_cookies
+                            output:
+                            - - classification: Other
+                                classification_id: 5
+                                default_milestone: '---'
+                                description: Test coverage for Red Hat Enterprise
+                                  Linux. Use for tracking bugs in automated and manual
+                                  test cases.
+                                disallow_new: false
+                                id: 182
+                                max_vote_super_bug: 10000
+                                milestone_url: https://wiki.test.redhat.com/BaseOs/Projects/TestsBugzilla
```

Comment:
It is already covered in README, just I've forgot to remove it automatically. it is possible theoretically do it as part of pre-commit, it it will be incorporated into